packages/commands/generate/src/index.ts

```diff
@@ -147,9 +147,20 @@ export const plugin: CliPlugin = {
           });
           const generateConfig: GenerateConfig = await config.extension('generate');
 
-          if (!generateConfig) {
+          if (!generateConfig && generateConfig.folders) {
```

Comment:
Also we need this kind of validation for each process, because for example resolvers generation only needs `folders.resolvers` etc. So for resolvers generation we need to check if `folders.resolvers` defined etc.